helm-chart/binderhub/values.yaml

```
@@ -197,3 +197,4 @@ initContainers: []
 extraVolumes: []
 extraVolumeMounts: []
 extraEnv: []
+annotations: {}
```

I suggest this because I've seen Helm chart typically differentiate between pod annotations and deployment annotations, and you are changing podAnnotations in this PR.

```suggestion
podAnnotations: {}
```

---

thanks @consideRatio for the review!

> Helm chart typically differentiate between pod annotations and deployment annotations

Where can I read about this?

---

Here is output from `helm create` in Helm v3.4.0. There isn't much documentation about this as a best practice or similar so its quite hard to follow the Helm chart trends in my mind. I have contributed changes to the `helm create` templates, and that in my mind is the best way to verify a practice is worthy as a best practice, along with getting changes merged to their documentation about best practices.

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{ include "tmp.fullname" . }}
  labels:
    {{- include "tmp.labels" . | nindent 4 }}
spec:
  {{- if not .Values.autoscaling.enabled }}
  replicas: {{ .Values.replicaCount }}
  {{- end }}
  selector:
    matchLabels:
      {{- include "tmp.selectorLabels" . | nindent 6 }}
  template:
    metadata:
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}
# ...
```

---

thanks!